bftengine/tests/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 set(CONFIG_FOLDER_PATH_VALUE "${CMAKE_CURRENT_SOURCE_DIR}/config")
-add_subdirectory(simpleStorage)
+add_subdirectory(config)
 add_subdirectory(simpleTest)
 add_subdirectory(simpleKVBC)
 add_subdirectory(simpleKVBCTests)
+add_subdirectory(simpleStorage)
```

Comment:
Did you intentionally move simpleStorage down? Tests depend on it, thus I moved it to the beginning in the past.

Reply:
reverted